### ISSUE TITLE
Pickup denominator popup in New Score wizard not accessible

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#27278]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

1. Open New Score wizard
2. Add an instrument or select a template
3. Next
4. Open Measures popup
5. Enable pickup
6. Tab to the denominator combobox box
7. Try to change the value

Result: cursor keys have no effect, space and Enter simply accept the values and close the popup.  It appears there is no way to set the denominator using the keyboard.

In current 4.5.1 nightly build, it seems this is the only control affected (that I know of).  But for 4.5, the time signature popup has the same problem.  In 4.4.4, both work.

This does seem strange as I'm not aware of any relevant change made between 4.5 and 4.5.1 (or, for that matter, between 4.4.4 and 4.5) but for me these behaviors are reproducible.  And it was reported and confirmed by several blind users on a MuseScore accessibility mailing list (with respect to 4.5).

### Supporting files, videos and screenshots

![Image](https://github.com/user-attachments/assets/a521a7dc-7feb-4c10-b81b-15511305133f)

### What is the latest version of MuseScore Studio where this issue is present?

OS: Debian GNU/Linux 12 (bookworm), Arch.: x86_64, MuseScore Studio version (64-bit): 4.5.1-250800404, revision: github-musescore-musescore-aa4cc17

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

Debian 12

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by DmitryArefiev** [comment]
@MarcSabatella Hi! I checked this on 4.5.2 nightly on Win10, and it looks fixed now. Can you recheck it on your side too please? 

https://github.com/musescore/MuseScore/actions/runs/14163824216

**Comment by DmitryArefiev** [comment]
@MarcSabatella Thanks! Let's close it then.

> It would be good to understand what actually fixed it, just as it would be to understand what broke it, since I really don't recall seeing any relevant changes and I'd hate to see it just sort of break again because we don't know what to avoid.


@mathesoncalum  suggesting it may have been resolved by one of these PRs #27379, but I didn't check it.

**Comment by DmitryArefiev** [comment]
Tested in **4.5.2 nightly** on Win10, Mac13.7.2. The issue **no longer occurs**. Closing

**Comment by MarcSabatella** [comment]
Oh yes, that's extremely plausible - I see changes there to focus policy etc.  Thanks for the pointer!
